.gitignore

```diff
@@ -109,3 +109,4 @@ venv.bak/
 # Data Science
 .DS_Store
 data/
+.dvc/
```

Comment:
you're ignoring the whole `.dvc`, but you have a file in  `.dvc` which is git recorded. Odd.